README.md

```diff
@@ -1,21 +1,67 @@
 # Common files for os-autoinst/os-autoinst and os-autoinst/openQA
 
-This repository is to be used as a
-[git-subrepo](https://github.com/ingydotnet/git-subrepo).
+This repository is to be used as a [git-subrepo]
+(https://github.com/ingydotnet/git-subrepo). See the instructions below in the
+[Git Subrepo Usage](#git-subrepo-usage) section
 
+All dependencies in this project are sourced from what's available in OpenSUSE
+Tumbleweed RPM repositories.
+
+For developers not using RPM the tumbleweed repositories, a `cpanfile` is
+provided and maintained in a best-effort basis.
+
+## Tooling offered
+
+This repo offers:
+
+* Linter configuration for perl projects: `.perltidyrc`, `.perlcriticrc`,
```

Comment:
I would like to avoid the word "Linter". Rather make it "Static code checks". How about that?